web/src/compose.js

```diff
@@ -43,6 +44,10 @@ export function clear_private_stream_alert() {
     $(`#compose_banners .${CSS.escape(compose_banner.CLASSNAMES.private_stream_warning)}`).remove();
 }
 
+export function is_preview_area_showing() {
```

Comment:
`is_preview_area_visible` would be a better name for this, to align with the `:visible` attribute that is being checked for.